_assets/css/_monokai.scss

```diff
@@ -0,0 +1,192 @@
+.highlight pre {
```

Comment:
Would the reason to have it as a dependency be for the case of syntax class names changing in the future? Otherwise, it's a very simple css file. I also tweaked a few of the colour values to better fit in with our theme colour scheme. Would it be better if I renamed the file, since it's not the monokai theme anymore?

Reply:
If this is modified then I see there's two options:

* Rename this to make it clear it's modified
* Include the original from a package and just override the styles as needed.